Clinical-Genomics/cgbeacon2
# 0-based coords and other improvements

This PR contains the following fixes: 
- Make coordinates 0-based (fix #56)
- Info endpoint returns 2 examples of queries (one SNV and one SV query)
- Remove check for presence of end coordinate in SV query (it's not needed and makes it too strict)
-  Improved the readme file

### How to test the 0-based coordinates change:
- execute `cgbeacon2 add demo`
- run the server: `cgbeacon2 run`
- In another terminal window, run this query (0-based):
```
curl -X POST   -H 'Content-Type: application/json'   -d '{"referenceName": "20",
  "variantType": "DUP",
  "start": 54963147,
  "referenceBases": "C",
  "assemblyId": "GRCh37",
  "includeDatasetResponses": "ALL"}' http://localhost:5000/apiv1.0/query
```
Variant should found

### How to test the query examples in the info endpoint:
- run the server: `cgbeacon2 run`
- In another terminal window, run this query:
`curl -X GET 'http://localhost:5000/apiv1.0/'`

Response should return the 2 query examples

### How to test that the SV query without end coordinate works:
- The query above should return exists:true

### Expected outcome:
The tests should be working

### Review:
